lib/rdkafka/consumer.rb

```diff
@@ -385,6 +385,37 @@ def seek(message)
       end
     end
 
+    # Lookup offset for the given partitions by timestamp.
+    #
+    # @param list [TopicPartitionList] The TopicPartitionList with timestamps instead of offets
```

Comment:
offsets (typo)

Reply:
Thanks @mensfeld. Fixed the typo.

Reply:
Thanks. I will get back to this PR in the upcoming weeks.